src/v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -220,9 +226,62 @@ const processEvent = async (metadata, message, destination) => {
     }
 
     Object.values(createdPayload).forEach((data) => {
-      response.push(responseBuilder(metadata, data, destination));
+      responses.push(responseBuilder(metadata, data, destination));
     });
-    return response;
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const { body, method, params, endpoint, headers } of responses) {
```

Comment:
I'd probably do something like `Promise.all` or `Promise.allSettled` instead of for-loop

Reply:
Used promise.all as we should reject the promises as soon as we get rejection of any of the events.